test/native/httpd/install.sh

```diff
@@ -49,6 +49,8 @@ if [ -f /tmp/$FILECONF ]; then
   echo "Include conf/$FILECONF" >> /usr/local/apache2/conf/httpd.conf
 fi
 
+echo -e "\nMaxConnectionsPerChild    16"  >> /usr/local/apache2/conf/httpd.conf
```

Comment:
Perhaps add a note for the future why this is required here?

Reply:
I added a rather lengthy note. Is it ok like that?

Reply:
Looks good, thanks